Patterns/Dutch/Dutch-DateTime.yaml

```diff
@@ -681,22 +681,30 @@ UnitMap: !dictionary
     decennia: 10Y
     jaren: Y
     jaar: Y
+    jr: Y
     maanden: MON
     maand: MON
+    mnd: MON
     weken: W
     week: W
     dagen: D
     dag: D
+    dgn: D
     uren: H
     uur: H
     u: H
     minuten: M
     minuut: M
     min: M
+    min.: M
     secondes: S
+    seconden: S
     seconde: S
     secs: S
     sec: S
+    kwartier: H
```

Comment:
Are these also necessary? How does English output the unit for similar cases?

Reply:
same as above